### PR TITLE
Feature/cpw 2 0#40 partner with us page

### PR DESCRIPTION
**Pull request by alex-novak**
https://trello.com/c/7bAE9Dxx

### CODE REVIEW

File: assets/js/scripts.js [hunk 1]
```diff
@@ -206,3 +206,34 @@ $('.hamburger-top').on('click', function() {
...
+  wH = $(window).height(),
+  wS = $(this).scrollTop();
+  console.log((hT-wH) , wS);
+  if (wS > (hT+hH-wH)){
+  $('.js-data-1').animate({ num: 200 - 20 }, {
+  duration: 2000,
+  step: function (num){
+  this.innerHTML = (num + 20).toFixed(0) + '+'
+}
+});
```
Comment (albertvolkman) [inline assets/js/scripts.js]: Please fix the formatting in this file.

File: assets/js/scripts.js [hunk 2]
```diff
@@ -206,3 +206,34 @@ $('.hamburger-top').on('click', function() {
...
+
+
+  $('.js-data-3').animate({ num: 300 - 20 }, {
+  duration: 2000,
+  step: function (num){
+  this.innerHTML = (num + 20).toFixed(0)
+}
+});
+}
+});
```
Comment (albertvolkman) [inline assets/js/scripts.js]: Please fix your editor to include newlines at EoF.

Comment (alex-novak) [reply assets/js/scripts.js]: done

File: assets/js/scripts.js (hunk 1 above)
Comment (alex-novak) [reply assets/js/scripts.js]: done

**Comment by albertvolkman** [comment]
@alex-novak, I just realized the size of these images... one is 11MB??? Even the smaller ones should absolutely be resized. Please do so.

**Pull request merged**